bpm/src/main/java/org/jboss/pnc/bpm/task/BpmBuildTask.java

```diff
@@ -73,7 +73,7 @@ protected Serializable getProcessParameters() throws CoreException {
     private BuildExecutionConfigurationRest getBuildExecutionConfiguration(BuildTask buildTask) {
 
         BuildConfigurationAudited buildConfigurationAudited = buildTask.getBuildConfigurationAudited();
-        String contentId = ContentIdentityManager.getBuildContentId(buildConfigurationAudited.getName());
+        String contentId = ContentIdentityManager.getBuildContentId(buildConfigurationAudited);
 
```

Comment:
Since ContentIdentityManager is about strings, I would rather pass in additional primitive parameter then a model object.

Reply:
I had the same thought, but later I changed my mind as it's in the model module, but I have no problem with changing it.